packages/framework/ai-collab/README.md

```diff
@@ -1,34 +1,194 @@
 # @fluid-experimental/ai-collab
 
-Utilities for using SharedTree with LLMs.
-This package aims to assist SharedTree-based apps that want to leverage LLMs.
+## Description
+The ai-collab client library makes adding complex, human-like collaboration with LLM's built directly in your application as simple as one function call. Simply pass your SharedTree and ask AI to collaborate. For example,
```

Comment:
Nit: [Markdown best practices](https://www.markdownguide.org/basic-syntax/#heading-best-practices) call for a blank line between headers and subsequent content. I also always recommend line-breaking along sentence boundaries to make future diffs easier to read (single line breaks are rendered as a space in vanilla and github-flavored Markdown).

```suggestion
## Description

The ai-collab client library makes adding complex, human-like collaboration with LLM's built directly in your application as simple as one function call.
Simply pass your SharedTree and ask AI to collaborate. For example,
```

Reply:
Updated in commit b700ced170a403db24c887b25ce4917392948b3c